### PR TITLE
Space Tourism Conference 2 pages create with html,css,javaScript

### PR DESCRIPTION
**Pull request by ehs9nino**
In this project I have applied all the learning during the first whole month of Microverse  program. Which helped me to make a proper information website. The techniques and learning I applied are as follows.

a. I used linters tools to write clean code & used proper workflow of git hub & used the proper documentation.
b. Used best practices of HTML, CSS & JavaScript. Created different folder and separate files and linked with the main program.
c. Specifically created one main page and one about page and used 4 CSS files for the media query purpose.
d. Used the event listeners in Java Scripts for mobile hamburger button and all.
e. Used javaScripts objects to bring data to the main page speakers section. 
f. Used hover menu & linked all buttons properly.
g. Used loom to record my final work.

Please let me know if there need to be some major change so I can finish this my very first Capstone project ASAP.
I would like to learn from your review, thanks!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    <section class="main-programme" id="programme">
+        <div class="main-programme-content">
+            <h3>Event Schedule</h3>
+            <div class="line1"></div>
+            <div class="programme-cards">
+                <div class="programme-card">
+                    <img src="./icons/a.png" alt="Registration & VIP reception">
+                    <h4>Registration & VIP Reception</h4>
+                    <p>Registration at Renaissance Los Angeles Airport Hotel. Timing is from 01:00pm to 03:00pm April 27,2022.</p>
+                </div>
```
Comment (Cathella) [inline index.html]: - [ ] Kindly make the cards wrapper an article to make your code more semantic and readable.
```
<article class="programme-card">
    <img src="./icons/a.png" alt="Registration & VIP reception">
    <h4>Registration & VIP Reception</h4>
    <p>Registration at Renaissance Los Angeles Airport Hotel. Timing is from 01:00pm to 03:00pm April 27,2022.</p>
</article>
```

File: index.html [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" media="screen and (min-width: 768px)" href="./css/style-desktop.css" />
+    <title>Space Tourism Conference</title>
+</head>
+<body>
+    <header></header>
```
Comment (Cathella) [inline index.html]: - [ ] Please do not leave the `header` tag empty. I think you can use it wrap the `nav` instead

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,342 @@
...
+}
+
+.speaker-img {
+  align-self: center;
+}
+
+.speaker-img > img {
+  width: 100%;
+}
+
```
Comment (Cathella) [inline css/style.css:199]: - [ ] Kindly add a checked background on the `speaker-card` before the image as seen in the original design.

image

**Review by Cathella — CHANGES_REQUESTED** [review]
## 🟡 Status: Changes Required 🛑

### Hi @ehs9nino 👋🏻, 

You have quite a nice presentation of your project and it looks very nice too 😍. I enjoyed going through it 💪🏻!
However, there are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## To Highlight 👏🏻

✅ No linter errors.
✅ You used correct Github Flow.
✅ Descriptive PR.
✅ The project contains a Home page and About page with a nice navigation.
✅ Your project design/styling is beautiful and clean.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css (hunk 3 above)
Comment (ehs9nino) [reply css/style.css:199]: @Cathella  thanks for the code review, I fixed the problems now.

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @ehs9nino, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Enjoy your weekend🎉🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**